### ISSUE TITLE
code expired ;)

### ISSUE THREAD

**Issue by fragtion** [sucresware/voicemeeter-activator-autoit#1]
Hey

Great discovery of the technique for bypassing the annoying popup... I've been using this too for some months

I notice that the code is no longer working, I'm guessing the developer patched something. How's it looking on your side & Any ideas ? XD I'm using 738299 right now and it's working for now but it will probably expire soon .. xD

**Comment by jakitun** [comment]
the activator is for voicemeeter v. 3.0.1.3
the current voicemeeter version is 3.0.2.1

I think that's why its not working?

**Comment by fragtion** [comment]
Yes. This technique no longer works on newer versions. Repo could probably be archived as registry patch is useless now. There's another patcher in the wild which works on newer versions but you'll have to search for it

**Comment by Mich107** [comment]
> Yes. This technique no longer works on newer versions. Repo could probably be archived as registry patch is useless now. There's another patcher in the wild which works on newer versions but you'll have to search for it
Can you help me find it lol. I've looked all around but the only thing that comes up is the VM website.

**Comment by ZloyTapochek** [comment]
> Yes. This technique no longer works on newer versions. Repo could probably be archived as registry patch is useless now. There's another patcher in the wild which works on newer versions but you'll have to search for it

Yes, please. At least give me a hint where to look. And then I'm looking for a very long time and I can't find :<

**Comment by Morples** [comment]
> 

Me as well

**Comment by xTCry** [comment]
@fragtion, do you still have the installer for version 3.0.1.3?

**Comment by fragtion** [comment]
> @fragtion, do you still have the installer for version 3.0.1.3?

http://web.archive.org/web/20200430083109/https://download.vb-audio.com/Download_CABLE/Voicemeeter8Setup_v3013.zip

**Comment by aniel300** [comment]
Where can i find the new patch please=

**Comment by Morples** [comment]
> > @fragtion, do you still have the installer for version 3.0.1.3?
> 
> http://web.archive.org/web/20200430083109/https://download.vb-audio.com/Download_CABLE/Voicemeeter8Setup_v3013.zip

I love you

**Comment by MDtheSimp** [comment]
Anyone got some info on that "other patcher in the wild" couldn't find anything

**Comment by Minerbug** [comment]
So, is the workaround to install an older version?
How can i block auto updates and internet connection?

**Comment by aniel300** [comment]
same here, have had to deleted new updated version few time, this last time blocked sw with firewall and am hoping it doesnt auto update on me

**Comment by tankirules** [comment]
I recommend 
https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb
for latest voicemeeter potato version.
Works for me.

**Comment by CodeSparksHQ** [comment]
> I recommend https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb for latest voicemeeter potato version. Works for me.

Hey bud sadly that link is not working, any chances there is another link ? I am not paying again twice for this tool.

**Comment by scorthyn** [comment]
> > I recommend https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb for latest voicemeeter potato version. Works for me.
> 
> Hey bud sadly that link is not working, any chances there is another link ? I am not paying again twice for this tool.

Hey, I also need the link from that. Do you happen to have it?

**Comment by Matthack7779** [comment]
> I recommend https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb for latest voicemeeter potato version. Works for me.

not working

**Comment by dudematthew** [comment]
> I recommend https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb for latest voicemeeter potato version. Works for me.

http://web.archive.org/web/20240922155238/https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb

**Comment by Topazez** [comment]
> > I recommend https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb for latest voicemeeter potato version. Works for me.
> 
> http://web.archive.org/web/20240922155238/https://gist.github.com/BluBb-mADe/f5faa900639fd0f94ab5f0886a41bafb

THANK YOU
